projects/plugins/crm/includes/ZeroBSCRM.GeneralFuncs.php

```diff
@@ -1553,4 +1553,61 @@ function jpcrm_youtube_url_to_video_id( $video_url ) {
 
 /* ======================================================
    / YouTube Helpers
-   ====================================================== */
\ No newline at end of file
+====================================================== */
+
+// These constants were created to simplify the process when this page gets
+// moved to a React-based approach. It was a choice to not refactor any further
+// than this.
+define( 'SETTING_SECOND_ADDRESS_ALWAYS_HIDE', 0 );
+define( 'SETTING_SECOND_ADDRESS_ALWAYS_SHOW', 1 );
+define( 'SETTING_SECOND_ADDRESS_ALLOW_CHANGE_VISIBILITY', 2 );
+
+/**
+ * This function returns the correct value for the second address visibility
+ * based on two distinct settings keys. We are using two different settings keys
+ * because the option to allow changing the visibility was added later, and we
+ * want to keep backwards compatibility. After all uses of the second address
+ * visibility are dealt with (e.g. in the Client Portal), a migration
+ * can be written to use only one setting key.
+ *
+ * @param array $settings The settings array.
+ * @return int The value for the visibility of the second address fields.
+ */
```

Comment:
Looks like we're missing a summary for the DocBlock ([source](
https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods))

```suggestion
/**
 * Get secondary address admin visibility.
 *
 * <Why we need this function>
 *
 * @param array $settings The settings array.
 * @return int The value for the visibility of the second address fields.
 */
```

Reply:
Addressed in e76b3a0.